framework-plugins/lisk-framework-chain-connector-plugin/test/unit/utils.spec.ts

```diff
@@ -12,7 +12,107 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { getActiveValidatorsUpdate } from '../../src/active_validators_update';
+import { BLS_SIGNATURE_LENGTH, cryptography } from 'lisk-sdk';
+import * as utils from '../../src/active_validators_update';
+import {
+	calculateActiveValidatorsUpdate,
+	getActiveValidatorsUpdate,
+} from '../../src/active_validators_update';
+import { ADDRESS_LENGTH, BLS_PUBLIC_KEY_LENGTH, HASH_LENGTH } from '../../src/constants';
+
+describe('calculateActiveValidatorsUpdate', () => {
+	const certificateValidatorsHash = cryptography.utils.hash(cryptography.utils.getRandomBytes(32));
+	const lastCertificateValidatorsHash = cryptography.utils.hash(
+		cryptography.utils.getRandomBytes(32),
+	);
+	const certificate = {
+		aggregationBits: Buffer.alloc(1),
+		blockID: cryptography.utils.hash(cryptography.utils.getRandomBytes(8)),
```

Comment:
just out of curiosity, why not `cryptography.utils.getRandomBytes(HASH_LENGTH)`?